docs/cli.md

```diff
@@ -0,0 +1,30 @@
+# Command Line Interface
+
+Once you have [installed the blueprint](./getting-started.md), you can use it from the CLI.
+
+You can either provide the path to a configuration file:
+
+```bash
+document-to-podcast --from_config "example_data/config.yaml"
+```
+
+Or provide values to the arguments directly:
+
+
+```bash
+document-to-podcast \
+--input_file "example_data/Mozilla-Trustworthy_AI.pdf" \
+--output_folder "example_data"
+--text_to_text_model "Qwen/Qwen2.5-1.5B-Instruct-GGUF/qwen2.5-1.5b-instruct-q8_0.gguf"
```

Comment:
If you dont provide an argument, does it take the value from the `config.yaml`?

Reply:
No, you need to pass `--from-config` to make it parse a file. If you don't provide that argument, it works as a regular entrypoint.
Also, if you pass `--from-config` the rest of the args are just ignored (I could improve the arg parsing to make this more explicit and fail in case you pass something that will be ignored)
I considered this a good approach to support both interfaces but keeping them separated.

Reply:
> Maybe the example should have all the arguments necessary?

Right below there is the full API reference of the entrypoint